src/components/CustomClient.js

```diff
@@ -24,7 +27,7 @@ class CustomClient extends React.Component {
   state = {
     toLoad: null,  // parsed json
     error:  null,
-    json:   '',
+    json:   ``,
```

Comment:
I feel like we should configure eslint to use backticks for string literals (I'm assuming that's possible).

Reply:
(this is a @todo, not blocking this PR)

Reply:
Definitely feel free to make an issue.